addons/web_editor/static/src/js/editor/odoo-editor/test/spec/editor.test.js

```diff
@@ -2297,6 +2353,101 @@ X[]
                     });
                 });
             });
+            describe('Nested editable zone (inside contenteditable=false element)', () => {
+                it('should remove the uneditable nesting zone if the last element of the nested zone is empty', async () => {
+                    await testEditor(BasicEditor, {
+                        contentBefore: unformat(`
+                            <div contenteditable="false">
+                                <div contenteditable="true">
+                                    <p>[]<br></p>
+                                </div>
+                            </div>
+                        `),
+                        stepFunction: deleteBackward,
+                        contentAfter: unformat(`
+                            <p>[]<br></p>
+                        `),
+                    });
+                });
```

Comment:
I'm not sure about this one. It feels weird to me to be able to alter the non-editable context that is _containing_ your selection. We might want to discuss the use case for this in 3D too.

EDIT: I had a second look at the spec and that looks to be precisely what is expected.